### PR TITLE
PDP-1017: Change pdp-distilbert-squad base image

### PR DESCRIPTION
**Pull request by dutsuwak**
* Updated default image to nvidia/cuda:11.8.0-base-ubuntu22.04
* Updated installation of Python dependency as base image does not have it
* Added new optional parameters to payload to allow truncation and padding (both enabled by default)

[PDP-1017] [AUT-125]

[PDP-1017]: https://pureinsights.atlassian.net/browse/PDP-1017?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AUT-125]: https://pureinsights.atlassian.net/browse/AUT-125?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by ricardo-leon-pureinsights — DISMISSED** [review]
This looks good. We have tested this on Autodesk's cluster and it works like a charm.

File: README.md [hunk 1]
```diff
@@ -100,6 +100,11 @@ Verb: `POST`
 
 Endpoint: `http://localhost:8080/predict`
 
+Optional parameters for the Paylod.
```
Comment (julean-pi) [inline README.md]: Payload

File: README.md [hunk 2]
```diff
@@ -100,6 +100,11 @@ Verb: `POST`
...
+* `"truncation"` : Truncates to the max lenght of the model. Defaults to `True`.
```
Comment (julean-pi) [inline README.md]: length

File: README.md [hunk 3]
```diff
@@ -100,6 +100,11 @@ Verb: `POST`
...
+* `"padding"` : Adds padding to comply with the min lenght of the model. Defaults to `True`.
```
Comment (julean-pi) [inline README.md]: length

**Review by julean-pi — CHANGES_REQUESTED** [review]

**Review by julean-pi — APPROVED** [review]

**Pull request merged**